metrics/events.go

```diff
@@ -28,7 +28,9 @@ func (m *Metrics) HandleFailureEvent(ctx context.Context, id types.RetrievalID,
 		_ = m.tempDataMap.Delete(id)
 		m.requestWithIndexerFailures.Add(ctx, 1)
 	case types.RetrievalPhase:
-
+		if storageProviderID != types.BitswapIndentifier {
```

Comment:
why not use the protocol() helper function here?

Reply:
protocol is a bit different. it does the string conversion to "graphsync" versus "bitswap"? I guess it doesn't matter.